middleware/middleware-1.3.8.md

```diff
@@ -0,0 +1,52 @@
+# fiskaltrust.Middleware 1.3.8 (Germany)
+This release of the Middleware includes the stabilized version of the DSFinV-K export and automatically backups the TSE's _.tar_ files on daily closing receipts. 
+
+<div class="alert alert-warning" role="alert" style="border-radius: 0">Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.</div>
+
+## Finalized feature: Local DSFinV-K export
+After publishing a preview version of our local DSFinV-K export in [version 1.3.6](middleware-1.3.6.md), we're happy to announce that this feature now leaves the preview state and is fully available as a stabilized, final functionality. We'd like to thank everyone who sent us their highly-valued feedback so far!
+
+Using this version enables POSOperators that use the free Middleware only (without any add-ons like the revision-safe cloud storage) to be fully compliant to the tax authorities' regulations. The cloud-version of our DSFinV-K export (which can be queried via the Portal and is included in our _POS Archive_) is of course up-to-date as well.
+
+The receipt case for getting a DSFinV-K export is `0x4445000000000002`, the returned file is a .zip stream. More details about how to access this endpoint can be found in our [docs](https://docs.fiskaltrust.cloud/doc/interface-doc/doc/general/function-structures/function-structures.html#journal-function).
```

Comment:
This should be journal type instead of receipt case IMO

Reply:
True, thanks!